projects/plugins/jetpack/tests/php/3rd-party/Domain_Mapping_Test.php

```diff
@@ -62,11 +62,11 @@ public function test_domain_mapping_should_not_try_to_hook_when_sunrise_disable(
 		// Both of these methods should not be called.
 		$stub->expects( $this->exactly( 0 ) )
 			->method( 'hook_wordpress_mu_domain_mapping' )
-			->will( $this->returnValue( false ) );
+			->willReturn( false );
```

Comment:
Why do we need a `willReturn` if it's never supposed to be called? What happens if it's not present?

Reply:
I think the check for "never called" will wind up throwing before it gets to trying to return anything, so the `->willReturn()` is probably pointless.